rtrlib/rtr/packets.c

```diff
@@ -154,6 +155,18 @@ struct pdu_reset_query {
 	uint32_t len;
 };
 
+struct pdu_aspa {
```

Comment:
Just to make sure, we don't need the attribute packed here?

https://stackoverflow.com/questions/5473189/what-is-a-packed-structure-in-c

Reply:
not sure either

Reply:
So I looked into it and this is fine. 

https://datatracker.ietf.org/doc/html/draft-ietf-sidrops-8210bis-10#name-aspa-pdu

Because they clearly spill everything out. (Look at the 16 bit zero field, so they take memory alignment already into consideration)

Reply:
that's what I've been thinking too, `pdu_router_key` needs the `packed` attribute because of the `ski` and `ski` fields